build.cmd

```diff
@@ -51,6 +51,7 @@ set RUNNER=".\packages\Microsoft.Net.Compilers.%CSI_VERSION%\tools\csi.exe"
 
 %RUNNER% .\artifacts\files\internal\runner.csx || goto :error
 %RUNNER% .\artifacts\files\internal\target-runner.csx || goto :error
+%RUNNER% .\artifacts\files\internal\target-runner-options.csx || goto :error

```

Comment:
This has to be a fixup to the second commit, not the first.

Reply:
I'm going home.

Reply:
Split.